src/styles/card.scss

```
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+}
+
+.card {
+  display: flexbox;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: $main-font-style;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px;
+    padding-bottom: 36px;
+    margin: 0;
+  }
+
+  &__title {
+    color: $dark-color;
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    padding-left: $padding-left;
+    text-align: left;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    color: $light-color;
+    font-size: 10px;
+    line-height: 14px;
+    padding-left: $padding-left;
+    padding-bottom: 16px;
+    margin: 0;
+  }
+
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  height: 18px;
+
+  &__title {
+    color: $light-color;
+    font-size: 12px;
+    line-height: 18px;
+    padding-left: $padding-left;
+    margin: 0;
+  }
+
+  &__cost {
+    color: $dark-color;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    padding-right: 16px;
+    margin: 0;
+  }
+}
+
+.buy {
+  display: flex;
+  width: 166px;
```

Use 100% everywhere and add padding for the card
```suggestion
  width: 100%;
```

---

then you should do 
```
max-width: 166px;
width: 100%
```